Add Enum for OS

MaxHorwood opened the pull request:
I think this could be an interesting project to help out on but looks pretty rough.

Showing how you might use Enums etc for this part / adding typing would be benifical.

Did vague testing but the saving / loading needs checking.

Commit:
Add Enum for OS

Commit:
ignore + missed strings

relt-1 commented:
its actually Mac OS 9 Windoid. It looks like this:
![image](https://user-images.githubusercontent.com/60782515/194917243-23de20a8-42c5-42fc-a570-4ec7da87dfb9.png)

Commit:
Fix saving/loading and windoid enum

MaxHorwood commented:
> its actually Mac OS 9 Windoid. It looks like this: ![image](https://user-images.githubusercontent.com/60782515/194917243-23de20a8-42c5-42fc-a570-4ec7da87dfb9.png)

Didn't know that was a thing. I've fixed that. Also tested the saving and loading which wasn't working but I've fixed now.
